### PR TITLE
CDK-lite-template now deploys. Bumped Console to 1.28.0

### PR DESCRIPTION
**Pull request by BStarmerSmith**
Fixed some linting errors around the code. 
Fixed an indentation error in our CDK-lite-template so it actually works now.
Added dedicated AWS CF linter cfn-lint.
Bumped Console to 1.28.0.

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: '3.8'
 
-services:  
-  
+services:
```
Comment (qboileau) [inline docker-compose.yml:3]: Can you also update console and cortex version here

**Comment by BStarmerSmith** [comment]
If you want to review you can for now, but I realised my linter wasn't fully working, I've fixed it locally but need to fix some CF issues before I can actually post it

**Review by qboileau — APPROVED** [review]
LGTM, we can also update console on compose file (even if right now I don't know why it's here in this repository :thinking: )

File: docker-compose.yml (hunk 1 above)
Comment (BStarmerSmith) [reply docker-compose.yml:3]: Sure thing!

**Pull request merged**